build_test.go

```diff
@@ -261,12 +261,15 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 			entryResolver.ResolveCall.Returns.BuildpackPlanEntry = packit.BuildpackPlanEntry{
 				Name: "dotnet-aspnetcore",
 				Metadata: map[string]interface{}{
-					"version-source": "buildpack.yml",
+					"version-source": "BP_DOTNET_FRAMEWORK_VERSION",
```

Comment:
The test ending on L257 need not be run with `spec.Sequential()` since the unit tests are now being run sequentially.